doc/monitoring/when_use_cartridge.rst

```diff
@@ -0,0 +1,221 @@
+доступны новые данные по этим вызовам:
+
+.. code-block:: json
+
+    {
+        "label_pairs": {
+            "path": "/hello",
+            "method": "ANY",
+            "status": 200,
+            "alias": "my-tnt-app"
+        },
+        "timestamp": 1679668258972227,
+        "metric_name": "http_server_request_latency_count",
+        "value": 9
+    },
+    {
+        "label_pairs": {
+            "path": "/hello",
+            "method": "ANY",
+            "status": 200,
+            "alias": "my-tnt-app"
+        },
+        "timestamp": 1679668258972227,
+        "metric_name": "http_server_request_latency_sum",
+        "value": 0.00008015199273359
+    },
+
+По умолчанию тип данной метрики - ``histogramm``, но можно использовать ``summary``. См.
+`здесь <http...>`__.
+
+.. _when_use_cartridge-advanced_settings:
+
+#.  Since Tarantool Cartridge ``2.4.0``, you can set a zone for each
+    instance in the cluster. When a zone is set, all the metrics on the instance
+    receive the ``zone`` label.
+
+#.  You can set custom global labels with the following configuration:
+
+    ..  code-block:: yaml
+
+        metrics:
+          export:
+            - path: '/metrics'
+              format: 'json'
+          global-labels:
+            my-custom-label: label-value
+
+    Another option is to invoke the ``set_default_labels`` function in ``init.lua``:
+
+    ..  code-block:: lua
+
+        local metrics = require('cartridge.roles.metrics')
+        metrics.set_default_labels({ ['my-custom-label'] = 'label-value' })
+
+#.  You can use the configuration below to choose the default metrics to be exported.
+    If you add the include section, only the metrics from this section will be exported:
+
+    ..  code-block:: yaml
+
+        metrics:
+          export:
+            - path: '/metrics'
+              format: 'json'
+          # export only vinyl, luajit and memory metrics:
+          include:
+            - vinyl
+            - luajit
+            - memory
+
+    If you add the exclude section,
+    the metrics from this section will be removed from the default metrics list:
+
+    ..  code-block:: yaml
+
+        metrics:
+          export:
+            - path: '/metrics'
+              format: 'json'
+          # export all metrics except vinyl, luajit and memory:
+          exclude:
+            - vinyl
+            - luajit
+            - memory
+
+    For the full list of default metrics, check the
+    :ref:`API reference <metrics-api_reference-functions>`.
+
+.. _when_use_cartridge-custom_health_handle:
+
+Свой формат healthcheck
+-----------------------
+
+По умолчанию ответ команды ``health`` содержит статус 200, если всё ок. И статус 500, если инстанс
+нездоров. Вы можете установить свой формат ответа следующим образом:
+
+.. code-block:: lua
+
+    local metrics = require('cartridge.roles.metrics')
+    local health = require('cartridge.health')
+
+    metrics.set_custom_is_health_handler(function(req)
```

Comment:
So ot seems that this patch is blocked by https://github.com/tarantool/cartridge/pull/2098 and https://github.com/tarantool/metrics/pull/450